src/Lock.php

```diff
@@ -987,6 +988,25 @@ public static function showMassiveActionsSubForm(MassiveAction $ma)
 
                 echo "<br><br>" . Html::submit(_x('button', 'Post'), ['name' => 'massiveaction']);
                 return true;
+            break;
+            case 'unlock_fields':
+                $related_itemtype = str_replace("massform", "", $ma->POST['container']);
```

Comment:
Should probably work.

```suggestion
                $related_itemtype = $ma->getItemtype();
```